code/pages/ExtensibleSearchPage.php

```diff
@@ -658,10 +665,26 @@ public function getForm($filters = true) {
 		}
 
 		// Construct the search form.
+		$search = TextField::create('Search', _t('SearchForm.SEARCH', 'Search'), isset($_GET['Search']) ? $_GET['Search'] : '')
+				->addExtraClass('extensible-search search')
+				->setAttribute('data-extensible-search-page', $this->data()->ID);
 
-		$fields = new FieldList(
-			TextField::create('Search', _t('SearchForm.SEARCH', 'Search'), isset($_GET['Search']) ? $_GET['Search'] : '')->addExtraClass('extensible-search search')->setAttribute('data-suggestions-enabled', Config::inst()->get('ExtensibleSearchSuggestion', 'enable_suggestions') ? 'true' : 'false')->setAttribute('data-extensible-search-page', $this->data()->ID)
-		);
+		if(Config::inst()->get('ExtensibleSearchSuggestion', 'enable_suggestions')) {
+			$search->setAttribute('data-suggestions-enabled',
+					Config::inst()->get('ExtensibleSearchSuggestion', 'enable_suggestions') ? 'true' : 'false'
```

Comment:
This may need to remain evaluated as a string, as javascript was previously having trouble using a boolean during the search suggestion implementation. This will likely pop up again if we keep the current search suggestions javascript.